src/views/WorkspaceDetail.vue

```diff
@@ -122,14 +136,20 @@
               :loading="loading"
             />
           </v-card>
-        </v-flex>
-      </v-layout>
+        </v-col>
+      </v-row>
     </v-main>
   </v-container>
 </template>
 
 <script lang="ts">
-import Vue, { PropType } from 'vue';
+import {
+  defineComponent, ref, PropType, computed, watch,
+} from '@vue/composition-api';
+
+// import { useRouter } from 'vue-router'; // Router import vue3 syntax
+// eslint-disable-next-line import/no-cycle
+import router from '@/router'; // Temporary router syntax that lets us use the old push
```

Comment:
```suggestion
```

If my suggestion around `setup` is committed, this is no longer needed.

Reply:
See c141af6